product_configurator_wizard/wizard/product_configurator.py

```diff
@@ -368,6 +433,26 @@ def fields_view_get(self, view_id=None, view_type='form',
         # Update result dict from super with modified view
         res.update({'arch': etree.tostring(mod_view)})
 
+        # set any default values
+        wiz_vals = wiz.read(dynamic_fields.keys())[0]
+        dynamic_field_vals = {
+            k: wiz_vals.get(
+                k, [] if v['type'] == 'many2many' else False
+                )
+            for k, v in fields.iteritems()
+            if k.startswith(self.field_prefix)
+        }
+        domains = {k: dynamic_fields[k]['domain']
+                   for k in dynamic_field_vals.keys()}
+        try:
+            cfg_step_id = int(wiz.state)
+            cfg_step = wiz.product_tmpl_id.config_step_line_ids.filtered(
+                lambda x: x.id == cfg_step_id)
+        except:
+            cfg_step = self.env['product.config.step.line']
+        vals = wiz.get_form_vals(dynamic_field_vals, domains, cfg_step)
```

Comment:
If you create a condition-free default, it will be set here but changes will not cascade.